src/addons/conditionwatchers/addonconditionwatchertriggertimesecs.cpp

```diff
@@ -20,21 +21,17 @@ AddonConditionWatcher* AddonConditionWatcherTriggerTimeSecs::maybeCreate(
 
 AddonConditionWatcherTriggerTimeSecs::AddonConditionWatcherTriggerTimeSecs(
```

Comment:
This one is also redundant with the other time condition watchers.

Reply:
Why? this is not a specific time. It's X seconds from the first activation of the client.

Reply:
Just because the new code looks like it's the same.